### PR TITLE
Update Tests projects target frameworks

### PR DESCRIPTION
**Pull request by fgiuliani**

### CODE REVIEW

File: lib/PuppeteerSharp.Tests/PuppeteerSharp.Tests.csproj [hunk 1]
```diff
@@ -1,26 +1,33 @@
...
     <IsTestProject>true</IsTestProject>
-    <LangVersion>7.3</LangVersion>
+    <LangVersion>8.0</LangVersion>
+    <ReleaseVersion>0.0.0</ReleaseVersion>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+    <DocumentationFile>bin\Debug\netcoreapp3.1\PuppeteerSharp.Tests.xml</DocumentationFile>
+    <NoWarn>1701;1702</NoWarn>
   </PropertyGroup>
```
Comment (kblok) [inline lib/PuppeteerSharp.Tests/PuppeteerSharp.Tests.csproj:13]: We won't need xml docs for tests in PuppeteerSharp

Comment (fgiuliani) [reply lib/PuppeteerSharp.Tests/PuppeteerSharp.Tests.csproj]: @kblok removed

**Comment by kblok** [comment]
It seems we need to change these CI settings:

 * https://github.com/hardkoded/puppeteer-sharp/blob/master/appveyor-edge.yml#L36
 * https://github.com/hardkoded/puppeteer-sharp/blob/master/appveyor-fullframework.yml#L12
 * https://github.com/hardkoded/puppeteer-sharp/blob/master/appveyor-linux.yml#L11
 * https://github.com/hardkoded/puppeteer-sharp/blob/master/appveyor.yml#L11-L12

**Comment by fgiuliani** [comment]
@kblok changed

**Comment by kblok** [comment]
Let’s close this for now

**Pull request closed without merging**